src/Types.hs

```diff
@@ -71,6 +71,8 @@ module Types
   , csChannels
   , csChannelSelectState
   , csEditState
+  , csClientConfig
+  , csUsers
```

Comment:
We underwent a recent refactor to avoid exporting `csUsers` and many other lenses. There are now various functions to get the underlying data in a way that conceals the underlying representation. Can you use those? In `Types`, see e.g. `allUserIds`, `usernameForUserId`, `userIdForUsername`, etc. - there's a long list of such accessors in the export list now. (Those are very new relative to your last pass at doing this work.)